app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html

```diff
@@ -54,7 +54,8 @@ <h2 class="heading-small heading-section">
 
   @if(groupField.repeatsMax.isDefined && !disableAddButton) {
     <button
-      type="submit"
+      type="button"
```

Comment:
Simon

But I think this prevents users from adding sections unless they have JavaScript enabled?

We have to retain the ability to complete the journey with JavaScript disabled